plugin/disks/CMakeLists.txt

```diff
@@ -1,7 +1,22 @@
 INCLUDE (CheckIncludeFiles)
-CHECK_INCLUDE_FILES ("sys/statvfs.h;mntent.h" INFO_HEADERS LANGUAGE CXX)
 
-IF (INFO_HEADERS)
+CHECK_SYMBOL_EXISTS (getmntent_r "mntent.h" HAVE_GETMNTENT_R)
+CHECK_SYMBOL_EXISTS (getmntent "mntent.h" HAVE_GETMNTENT)
+CHECK_SYMBOL_EXISTS (getmntinfo "sys/mount.h" HAVE_GETMNTINFO)
+CHECK_SYMBOL_EXISTS (getmntinfo64 "sys/mount.h" HAVE_GETMNTINFO64)
+
```

Comment:
I had to change sys/mount to sys/types;sys/mount.h for getmntinfo, OpenBSD. You should probably do the same for getmntinfo64.

Reply:
Thanks. There's a `.h` on types too right `sys/types.h;sys/mount.h`? Does the plugin show decent data? config: `plugin-load-add=disks` and `select * from information_schema.disks'

Reply:
```
CHECK_SYMBOL_EXISTS (getmntinfo "sys/types.h;sys/mount.h" HAVE_GETMNTINFO)
```

Reply:
fixed